Add powered-by footer to event list, alternative to #63

dmfs opened the pull request:
@lemonboston please review. We're going to add this to the list for now, but we're certainly going to add some behavior later on.

dmfs committed:
Add powered-by footer to event list, alternative to #63

lemonboston commented on eventdiscovery-sdk/src/main/res/layout/schedjoules_event_list.xml:
```diff
@@ -12,9 +12,27 @@
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
                 app:layoutManager="LinearLayoutManager"
+                android:layout_marginBottom="20dp"
```
I checked with layout inspector and this margin leaves a little bit too much space covered by the footer, can be changed to 24dp safely (I checked), to avoid incorrect proportions of the bottom items (no more events).

lemonboston commented on eventdiscovery-sdk/src/main/res/layout/schedjoules_event_list.xml:
```diff
@@ -12,9 +12,27 @@
         <include layout="@layout/sticky_header_layout"/>
+
+
+        <com.schedjoules.eventdiscovery.framework.microfragments.eventdetails.fragments.views.SchedJoulesFooterView
```
I think the duplication should be resolved with the footer in the details view, either with layout of its own and `include` tag (extra tags can be added to `include`), or extracted style, but probably `include` is better, imo.

dmfs commented:
@lemonboston please review again. Note that I've rebased and squashed the commits, so you should remove your local branch first.
The problem was that the text size wasn't set to 0 which means the bar was slightly larger than the image+padding. Setting the textSize to 0 and increasing the padding fixed it.

lemonboston commented:
Why is `SchedJoulesFooterView` still a `TextView`? Couldn't it be simply an `ImageView`?

I think it would be useful to move the common layout definition of `<SchedJoulesFooterView....>` to a separate layout file and include it to remove the duplication. And use it like:
```
<include layout='schedjoules_powered_by_footer'
         app:layout_anchor="@id/schedjoules_footer_placeholder"
        app:layout_anchorGravity="bottom|center_horizontal"/>
```
So adding the special properties at the place of include.

dmfs commented:
If we use an `ImageView` we have to wrap it in a `FrameLayout` to get the padding and background color right. I agree that a `TextView` looks strange but it serves the purpose.
Good idea with the `include`. I'll do that.
